include/dlaf/tune.h

```diff
@@ -21,6 +21,11 @@ namespace dlaf {
 /// DLA-Future tuning parameters.
 ///
 /// Holds the value of the parameters that can be used to tune DLA-Future.
+/// - debug_dump_trisolver_data:
+///     Enable dump of trisolver input/output data to "trid-ref.h5" file that will be created in the
+///     working folder (it should not exist before the execution).
+///     WARNING: just a single execution can be dumped on disk, and any subsequent call fails.
+///     Set with environment variable DLAF_DEBUG_DUMP_TRISOLVER_DATA.
```

Comment:
Maybe it makes sense to put this somewhere else given that it's:
1. not a tuning parameter
2. a debugging parameter
? Not a blocker for me, but it would avoid having to put a warning public documentation.

Reply:
It is a good point.

@rasolca what do you think? probably not for this PR, but we can track this idea with an issue if you agree with @msimberg

Reply:
Just created #918